unittest/resources/test_user.py

```diff
@@ -26,14 +27,196 @@ def test_something(self):
+        db.session.add(user1)
+        db.session.add(user2)
+        db.session.commit()
+        resp = self.client.get('/users')
+
+        assert resp.status_code == 200
+        assert json.loads(resp.data) == {
+            'info': {},
             'status': {
                 'statusMsg': 'Ok',
                 'statusDetails': {},
                 'statusCode': 'HTTPOK'
             },
+            'result': [{
+                'id': 1,
+                'name': 'Test1 User',
+                'email': 'test1@test.com'
+                },
+                {
+                'id': 2,
+                'name': 'Test2 User',
+                'email': 'test2@test.com'
+                }]
+        }
+
+
+class TestUser_post(TestUser):
+    """Test resources for the API post endpoint.
+    """
+    def test_post_success(self):
+        user = {
+            'name': 'Test User',
+            'email': 'test@test.com',
+            'password': 'test'
+            }
+
+        resp = self.client.post('/users/', data=user)
+
+        assert resp.status_code == 200
+        assert json.loads(resp.data) == {
+            'info': {},
+            'status': {
+                'statusCode': 'HTTPOK',
+                'statusMsg': 'Ok',
+                'statusDetails': {}
+            },
+            'result': {
+                'id': 1,
+                'name': 'Test User',
+                'email': 'test@test.com'
+            }
+        }
+
+    def test_post_badRequest(self):
+        user = {
+            'name': None,
+            'email': None,
+            'password': None
+        }
+
+        resp = self.client.post('/users/', data=user)
+
+        assert resp.status_code == 400
+        assert json.loads(resp.data) == {
+            'info': {},
+            'status': {
+                'statusCode': 400,
+                'statusMsg': 'Bad request',
+                'statusDetails': {}
+            },
+            'result': {}
+        }
+
+
+class TestUser_put(TestUser):
+    """Test resources for the API put endpoint.
+    """
+    def test_put_success(self):
+        user = User('Test User', 'test@test.com', 'password')
+        db.session.add(user)
+        db.session.commit()
+
+        new_user_data = {
+            'name': 'Updated User',
+            'password': 'updatedpass'
+        }
+
+        resp = self.client.put('/users/%s' % user.id, data=new_user_data)
+
+        assert resp.status_code == 200
+        assert json.loads(resp.data) == {
+            'info': {},
+            'status': {
+                'statusCode': 'HTTPOK',
+                'statusMsg': 'Ok',
+                'statusDetails': {}
+            },
             'result': {
                 'id': 1,
-                'name': 'Test User'
+                'name': 'Updated User',
+                'email': 'test@test.com'
             }
         }
+
+    def test_put_notFound(self):
+        """Verify NotFound is raised when userId does not exist
```

Comment:
I think it is more accurate to say. `Verify NotFound is raised when a user with the given ID does not exist.`